e2e_tests/tests/per_provider/normal_tests/export_public_key.rs

```diff
@@ -210,3 +201,37 @@ fn check_export_ecc_public_possible() -> Result<()> {
 
     Ok(())
 }
+
+#[cfg(not(any(feature = "pkcs11-provider", feature = "tpm-provider")))]
+#[test]
+fn import_and_export_ecc_public_key() -> Result<()> {
```

Comment:
Feel free to ignore this since it isn't in the remit of this PR, but it would be nicer if these tests panicked instead of returning a `Result`.

Reply:
Yes agree, there are other tests where it's like that. It would be nice to make one PR that changes everything. Added #444 for that.